tests/config/test_configtypes.py

```diff
@@ -2087,41 +2116,43 @@ def test_validate_empty_none_ok(self):
 
     def test_validate_empty_item(self):
         """Test validate with empty item and none_ok = False."""
-        with self.assertRaises(configexc.ValidationError):
+        with pytest.raises(configexc.ValidationError):
             self.t.validate('foo,,bar')
 
     def test_validate_empty_item_none_ok(self):
         """Test validate with empty item and none_ok = True."""
         t = configtypes.UrlList(none_ok=True)
-        with self.assertRaises(configexc.ValidationError):
+        with pytest.raises(configexc.ValidationError):
             t.validate('foo,,bar')
 
     def test_transform_single(self):
         """Test transform with a single value."""
-        self.assertEqual(self.t.transform('http://qutebrowser.org/'),
-                         [QUrl('http://qutebrowser.org/')])
+        actual = self.t.transform('http://qutebrowser.org/')
+        expected = [QUrl('http://qutebrowser.org/')]
+        assert actual == expected
 
     def test_transform_more(self):
         """Test transform with multiple values."""
-        self.assertEqual(
-            self.t.transform('http://qutebrowser.org/,http://heise.de/'),
-            [QUrl('http://qutebrowser.org/'), QUrl('http://heise.de/')])
+        actual = self.t.transform('http://qutebrowser.org/,http://heise.de/')
+        expected = [QUrl('http://qutebrowser.org/'), QUrl('http://heise.de/')]
+        assert actual == expected
 
     def test_transform_empty(self):
         """Test transform with an empty value."""
-        self.assertEqual(self.t.transform(''), None)
+        assert self.t.transform('') is None
```

Comment:
You're correct, but it is recommended to compare against `None` using the `is` operator. In fact I believe this was supposed to be `self.assertIsNone(self.t.transform(''))`, so I went ahead and changed it. :smile: